nixos/modules/tasks/network-namespaces.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I wasn't sure whether to put this in it's own file or in `network-interfaces.nix`. Happy to move it there if people prefer :slightly_smiling_face:

Reply:
Unless I'm misunderstanding `tasks/network-interfaces-systemd.nix` is explicitly for `networkd` related options, so I think this does not belong there.